### DISCUSSION TITLE
Project HÆLTH Blog #6

### DISCUSSION THREAD

**Discussion by SidMllr** [ldcdorn/haelth#58] [General]
# Project HÆLTH 🌱
This week our team worked on the extension of our first demo from last week.
## Our selected user story
We chose to complete one of our user stories this week, since our work and goals heavily rely on those.
The following one was selected:

**User story  https://github.com/ldcdorn/haelth/issues/6: As a User and Athlete, I want an interface that is easy to use, clearly structured, and visually appealing which I can quickly and effortlessly access, without being overwhelmed by complex features or cluttered designs.**

**"Done"-criteria:**
- The interface must provide an intuitive and easy-to-navigate menu structure
- The design should follow a minimalist approach
- Buttons, icons, fonts, and color schemes must be consistent throughout the interface to ensure a uniform and seamless user experience.

We chose this one, since the UI is the base of our app and we could easily split the workload between our team members. Also, we wanted to have a big part finished right at the start of the project, and this user story was labeled as size "L".

## The second demo
Since our first demo, a lot has changed! We now have a UI for all three key sections of our app: Mindfulness, Nutrition & Fitness.
Let's start off with Mindfulness:

![mind](https://github.com/user-attachments/assets/8f8c44a4-a7a8-49d5-ada3-9ca2a92c1059)
As you can see, we implemented the UI for some basic meditation tips and also a stopwatch for custom meditation length.

At this point, we can explain the different layers of our .xml files. Since the three section-buttons at the bottom are always visible, they are part of the activity_main.xml. This means, that this layout will always be on top of the main content in the site. For the section of Nutrition, we want to show you only the fragment_dashboard.xml, excluding the other parts from activity_main.xml:

![grafik](https://github.com/user-attachments/assets/97a64b60-5f61-4823-9029-0f29ac6abcd0)
This shows the UI when adding a new meal, containing its name and the most important macro-nutrients.

![grafik](https://github.com/user-attachments/assets/e8c67be6-f0fc-4865-ac94-96a7d837e928)
We chose a similar UI-Layout for adding a new exercise in the Fitness-section, containing its name, the weight, repetitions and sets.

Overall, we are excited to have finished our first user story and the extension of our demo and are looking foreward to implementing more and more features in the following weeks!

**Comment by themsgcentre** [comment]
Wir finden, dass eure Fortschritte beeindruckend sind! Die Umsetzung einer minimalistischen und klar strukturierten Benutzeroberfläche ist euch gut gelungen. Besonders gefällt uns die Konsistenz im Design, wie die einheitliche Farbgebung und die einfache Navigation zwischen den Bereichen. Die Mindfulness-Funktion mit den Tipps und dem Timer wirkt funktional und einladend, und die Formulare für Ernährung und Fitness sind übersichtlich und praktisch gestaltet.

Es ist toll zu sehen, wie ihr die User Story priorisiert und den Fokus auf die Basis der App gelegt habt. Wir freuen uns darauf zu sehen, wie ihr die weiteren Features umsetzt und die App weiterentwickelt. Bemerkenswerte Arbeit – weiter so!

Viele Grüße,
Projekt TDC (HM)
